spring-rabbit/src/test/java/org/springframework/amqp/rabbit/listener/BlockingQueueConsumerTests.java

```diff
@@ -113,6 +113,9 @@ public void testPrefetchIsSetOnFailedPassiveDeclaration() throws IOException {
 
 		when(connectionFactory.createConnection()).thenReturn(connection);
 		when(connection.createChannel(Mockito.anyBoolean())).thenReturn(channel);
+		com.rabbitmq.client.Connection rabbitConnection = mock(com.rabbitmq.client.Connection.class);
+		when(rabbitConnection.isOpen()).thenReturn(true);
+		when(channel.getConnection()).thenReturn(rabbitConnection);
```

Comment:
`when(channel.isOpen()).thenReturn(true);` instead.
And:
```
blockingQueueConsumer.setRetryDeclarationInterval(100);
blockingQueueConsumer.setFailedDeclarationRetryInterval(100);
```
to make this test quick as before.